docs/ModelStore.md

```diff
@@ -27,12 +27,13 @@ The new page shows models available in model stores.
 
 ![List models](images/model-store-list.png)
 
-Hover the Note field to show complete text.
+Hover on Note will show the complete notes.
 Enter keyword in 'Filter list by' to limit results.
 Click 'Update model list' button will retrieve the latest model list (see limitation).
-Click 'Import' will import that model into DIGITS (may takes a few seconds, depends on network speed).
+Click 'Import' icon will import that model into DIGITS (may takes a few seconds, depends on network speed).
```

Comment:
How about:

Click "Import" **to** import that model into DIGITS (may **take** a few seconds, **depending** on network speed).